README.md

```diff
@@ -35,6 +35,16 @@ end
 
 ![Hello_World](assets/hello_world.png "Hello World sample")
 
+Alternatively, you could use [Glimmer DSL for WX](https://github.com/AndyObtiva/glimmer-dsl-wx) on top of wxruby3.
```

Comment:
Could you move the alternative Glimmer samples to the Glimmer FAQ section? Maybe under a subheading like "Using Glimmer for WX the examples above could be written like:" ?

Reply:
I removed the Glimmer examples and moved them to a [Coming From wxruby3](https://github.com/AndyObtiva/glimmer-dsl-wx#coming-from-wxruby3) section in the Glimmer DSL for WX README. I mentioned that section the FAQ item about Glimmer DSL for WX.